### DISCUSSION TITLE
انتخاب برنامه به عنوان برنامه دیفالت

### DISCUSSION THREAD

**Discussion by mohammad-noohi** [amir1376/ab-download-manager#73] [Q&A]
سلام خسته نباشید ،
یه سوالی داشتم میخواستم بدونم چجوری میشه این نرم افزار رو به عنوان نرم افزار دیفالت توی ویندوز قرارش داد چون هرچی گشتم چیزی برام نیورد

**Comment by amir1376** [comment]
@mohammad-noohi 
درود بر شما. وقتون بخیر
منظورتون بعنوان نرم افزار دیفالت چی هست؟ ( چه سناریویی مد نظرتونه؟ )

 اگر میخواین که از توی مرورگر فایل ها از طریق برنامه دانلود بشن افزونه مرورگر رو نصب کنین که لینکش توی [سایت](https://abdownloadmanager.com) موجوده
